week1/project/ecommerce/src/components/CategoryList.jsx

```diff
@@ -0,0 +1,21 @@
+export default function CategoryList({
+  list,
+  onCategoryClick,
+  selectedCategory,
+}) {
+  return (
+    <div>
```

Comment:
If this `div` does not have any stylistic reasons to be here (meaning that it needs to be here because of some CSS, like it's a child of a flex parent) then you can also use a "React Fragment". This will make sure you don't render unnecessary `div`s on the page 😉 

```suggestion
    <>your category list code here</>
```

Reply:
I used <div> because I might want to add some style in the future. But I think I can add the div if I want to do this.